__tests__/lib/__snapshots__/createGHA.test.ts.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`#createGHA command inputs changelogs should run GHA creation workflow and generate valid workflow file 1`] = `
+exports[`#createGHA command inputs changelogs (single) should run GHA creation workflow and generate valid workflow file 1`] = `
```

Comment:
These are just the resulting snapshot changes from splitting up the describe blocks above. No actual content changes here.